pkg/kubelet/cm/dra/plugin/plugin.go

```diff
@@ -132,7 +181,7 @@ func (h *RegistrationHandler) validateVersions(
 		)
 	}
 
-	existingPlugin := draPlugins.Get(pluginName)
+	existingPlugin := draPlugins.get(pluginName)
```

Comment:
Can we change this logic to the following to reduce the nesting levels and make it more clear in general:
```
	existingPlugin := draPlugins.get(pluginName)
	if existingPlugin == nil {
		return newPluginHighestVersion, nil
	}
	if existingPlugin.highestSupportedVersion.LessThan(newPluginHighestVersion) {
		return newPluginHighestVersion, nil
	}
	return nil, errors.New(
		log(
			"%s for DRA plugin %q failed. Another plugin with the same name is already registered with a higher supported version: %q",
			callerName,
			pluginName,
			existingPlugin.highestSupportedVersion,
		),
	)

Reply:
done in b55b0297bad